src/components/AnonymousChat/AnonPostNotificationChannelItem.tsx

```diff
@@ -16,6 +16,7 @@ const AnonPostNotificationChannelItem: (props: MessageChannelItemProps) => React
   item,
   onChannelPressed
 }) => {
+  // TODO: Change this into useUserAuthHook
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The comment suggests replacing `useProfileHook` with `useUserAuthHook`. If `useUserAuthHook` provides the same or more information and is more up-to-date or efficient, it would be a good idea to replace `useProfileHook` with it. However, without seeing the implementation of `useUserAuthHook`, I can't confirm if this change is necessary or beneficial. Please ensure that `useUserAuthHook` provides all the necessary data before making this change.
```diff
-  const {anonProfileId, signedProfileId} = useProfileHook();
+  const {anonProfileId, signedProfileId} = useUserAuthHook();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->